packages/fhir-router/src/repo.ts

```diff
@@ -363,6 +363,7 @@ export class MemoryRepository extends BaseRepository implements FhirRepository {
   }
 
   withTransaction<TResult>(callback: (client: unknown) => Promise<TResult>): Promise<TResult> {
+    // MockRepository currently does not support transactions
```

Comment:
Should we log something to console too maybe, since it could be a red herring pointing to a missing client / potentially silently "succeed"?